### PR TITLE
Blog post video 2019

### PR DESCRIPTION
**Pull request by yabellini**
A blog post about the end-year video 2019

### CODE REVIEW

**Comment by chucheria** [comment]
Please, merge the post even if the preview was not generated. This CI is independent from the master CI, some changes were done in master but not in preview. Thanks!

File: content/post/RLadies_video_2019.Rmd [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+# for their rows in one tibble/list using the map_dfr function of the purrr package
+
+eventos_todos_juntos <- eventos$result[!eventos_con_datos] %>% 
+  map_dfr(~ .) 
+
+# We then count the number of events held per year
+
+eventos_todos_juntos %>%
+  group_by(year(time)) %>%
+  summarise(cantidad=n())
```
Comment (ChrisZasa) [inline content/post/RLadies_video_2019.Rmd]: summarise(cantata = n())

File: content/post/RLadies_video_2019.Rmd [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+
+### Text, images ... audio?
+
+Now, a video with only letters, numbers, and images would leave many people out of our message, so we decided to record the audio of the message. For that, we got the help from the genius of [Alejandra Bellini](https://twitter.com/AlejaBellini), who recorded the audio in Spanish and English. She recorded it using WhatsApp with a cell phone, then we used [Zamzar](https://www.zamzar.com) to transform the audio to MP3 and [Mp3cut](https://mp3cut.net/en/) to cut the audio in the parts necessary to be able to synchronize the audio with the text and the video images.
+
+
+## Third step: editing ...
+
+_Third step_: with this plan in mind, it was time to edit the video. We did it using Doodly for the drawing-hand effects. Both music and fonts are provided by the software. The most laborious part was the synchronization of the audio with the drawing of the numbers and letters.
```
Comment (ChrisZasa) [inline content/post/RLadies_video_2019.Rmd]: We used the software Doodly, providing music and fonts, for the drawing-hand effects.

File: content/post/RLadies_video_2019.Rmd [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+# Getting all the R-Ladies groups
+all_rladies_groups <- find_groups(text = "r-ladies")
+
+# Cleaning the list
+rladies_groups <- all_rladies_groups[grep(pattern = "rladies|r-ladies", 
+                                          x = all_rladies_groups$name,
+                                          ignore.case = TRUE), ]
+```
+
+With the groups´ list, we search for all the events carried out by each of these groups and calculate the number of events:
```
Comment (ChrisZasa) [inline content/post/RLadies_video_2019.Rmd]: groups' ?

File: content/post/RLadies_video_2019.Rmd [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+
+R-Ladies' story would be to highlight global achievements _with numbers_ (after all we are people who are dedicated to #rstats and R-Ladies numbers are wonderful) and _to thank those who accompanied us_.
+
+## Generating the script and the raw material for the video
+
+The second step is to assemble the script for the video, which will have three scenes.
+
+### Scene one: the global reach
+
+Numbers for global R-Ladies chapters are shown in a way to highlight how we organize ourselves. We also show what was done and how much we grew in 2019. To be able to express that idea we chose _the number of chapters, the number of countries, the number of members, and the total number of events held in the year 2019_, because these are concrete indicators. The first three numbers were extracted from the [R-Ladies Community Board](https://benubah.github.io/r-community-explorer/rladies.html). The number of events was calculated using the [meetupr](https://github.com/rladies/meetupr) package, developed by R-Ladies.
```
Comment (ChrisZasa) [inline content/post/RLadies_video_2019.Rmd]: we also visualise / illustrate what was done...

**Pull request merged**